include/openmc/distribution.h

```diff
@@ -13,6 +13,7 @@
 #include "openmc/memory.h" // for unique_ptr
 #include "openmc/vector.h" // for vector
 
+
```

Comment:
Probably don't need this extra line - in fact, it may casuse the clang formatting test in CI to fail.